summarize/src/analysis.rs

```diff
@@ -171,7 +171,9 @@ pub fn perform_analysis(data: ProfilingData) -> Results {
                     record_event_data(
                         &current_top.label,
                         &|data| match &current_top.event_kind[..] {
-                            QUERY_EVENT_KIND | GENERIC_ACTIVITY_EVENT_KIND => {
+                            QUERY_EVENT_KIND
+                            | GENERIC_ACTIVITY_EVENT_KIND
+                            | INCREMENTAL_HASH_RESULT_EVENT_KIND => {
```

Comment:
I took another look and I think we need to handle this the same way that we handle `INCREMENTAL_LOAD_RESULT_EVENT_KIND` below, just with `data.incremental_hashing_time -= current_event_duration;`. Why? Because in this step we take care of adjusting the timings of the thing currently on the stack. If a `INCREMENTAL_HASH_RESULT_EVENT_KIND` is on the stack it means we are within hashing something. If we now encounter something within that event (like the nested "intern-the-dep-node" event) then we don't want to attribute that to the hashing time.